src/editor/bin/Oni2.re

```diff
@@ -131,10 +136,21 @@ let init = app => {
       w.onKeyDown,
       event => {
         let _ =
-          switch (event.key) {
-          | Key.KEY_BACKSPACE => ignore(neovimProtocol.input("<BS>"))
-          | Key.KEY_ENTER => ignore(neovimProtocol.input("<CR>"))
-          | Key.KEY_ESCAPE => ignore(neovimProtocol.input("<ESC>"))
+          switch (event.key, event.shiftKey, event.ctrlKey) {
+          | (Key.KEY_TAB, true, _) =>
```

Comment:
Nice! Thanks for adding some of these to help out with #34 - definitely needed to test this functionality 👍